syncd/Syncd.cpp

```diff
@@ -53,7 +59,7 @@ Syncd::Syncd(
     m_vendorSai(vendorSai),
     m_veryFirstRun(false),
     m_enableSyncMode(false),
-    m_timerWatchdog(30 * 1000000) // watch for executions over 30 seconds
+    m_timerWatchdog(30 * 1000000 * ASAN_DELAY_FACTOR) // watch for executions over 30 seconds
```

Comment:
I think that comment should be updated as well to described what is ASAN_DELAY_FACTOR

Reply:
And maybe rename ASAN_DELAY_FACTOR -> DELAY_FACTOR so it won't confuse people in cases when ASAN is not enabled